afteraction-share.html

```diff
@@ -200,6 +200,38 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
 		</div>
 	</div>
 	{% endif %}
+
+	{% if page.custom_fields.share_whatsapp_message %}
+		<script>
+		var whatsappMsg = '{{ page.custom_fields.share_whatsapp_message }} ';
+		</script>
+	{% else %}
+		<script>
+		var whatsappMsg = '{{ page.followup.share_title_value }} ';
+		</script>
+	{% endif %}
+
+	<script>
+		jQuery(document).ready(function($){
+			$('.whatsapp-share').on('click', function(e) {
+				e.preventDefault();
+				// Set Whatsapp Share URL based on response from API
+				var getShareURL = '/share/link?type=ot&action_id={{action.id}}&short=1&page_name={{page.name}}';
+				$.ajax({
+					method: 'GET',
+					url: getShareURL,
+				}).done(function(shareURL) {
+					var currWhatsappHref = $(e.target).attr('href');
+					var newURL = 'https://api.whatsapp.com/send?text=' + whatsappMsg + shareURL;
+					$(e.target).attr('href', newURL);
+					var win = window.open(newURL);
+					if (win) {
+						win.focus();
+					}
+				});
```

Comment:
would it be valuable to have a fail condition handler here like what exists for the manual copy link implementation also in this PR? IMO it may not be necessary here so just noting this.

Reply:
sure! I just updated with a fail condition handler